### ISSUE TITLE
Upgrade version of rclone

### ISSUE THREAD

**Issue by drohm** [teejee2008/polo#155]
Is it possible to upgrade the version of rclone that Polo is using to this latest test version?  This version adds webdav support which allows connecting to Owncloud/Nextcloud hosts.

https://github.com/ncw/rclone/issues/580#issuecomment-335199069

In his comment, he provides a link to the test builds (linux included).

**Comment by drohm** [comment]
This might just be a case of updating the version of rclone on my machine, correct?

**Comment by teejee2008** [comment]
Yes, Polo will use whichever version is installed on your machine

**Comment by drohm** [comment]
Won't the UI need to change to accommodate WebDAV?

![polo-cloud-menu-account-type](https://user-images.githubusercontent.com/184959/31349914-0da0772e-acf3-11e7-970c-0c7ef531a89e.png)

**Comment by teejee2008** [comment]
You can type `rclone config` in a terminal to add the webdav account - no need to use the GUI

**Comment by teejee2008** [comment]
It's not tested though. Restart Polo after adding the account and let me know if you are able to browse the storage.
